starter-code/js/index.js

```diff
@@ -1,17 +1,43 @@
 function deleteItem(e){
-
+// var item = document.getElementByClass('wrapper');
+e.target.parentElement.parentElement.remove();
 }
 
-function getPriceByProduct(itemNode){
+// var input = document.querySelectorAll[i]('input');
+// input.addEventListener("keyup",)
 
-}
+// function getPriceByProduct(itemNode){
+//   var productCost = document.querySelectorAll('.cost');
+//   var productCount = document.querySelectorAll('.quantity');
+//   var grandTotal = parseInt(productCost[0].innerText) * parseInt(productCount[0].value);
+//   var total = document.querySelectorAll('.total');
+//   total[0].innerText = grandTotal.toFixed(2);
+// }
 
 function updatePriceByProduct(productPrice, index){
 
 }
 
 function getTotalPrice() {
+  // var start = 0;
+  // var productIndex = productCost.indexOf
+  var sumPrice = 0;
+
+for (var i=0; i < document.querySelectorAll('.product-name').length; i++) {
```

Comment:
Instead of searching things in the document inside an iteration, you can create a variable with an array, and iterate over this array (it will make the iteration faster).